src/features/game/types/crops.ts

```diff
@@ -13,6 +13,7 @@ export type CropName =
   | "Wheat";
 
 export type Crop = {
+  itemType: "food" | "tool" | "nft" | "crop" | "seed"
```

Comment:
This is a bit strange also. Should a developer be able to type a crop as a `tool`? This could leave to some unintended side effects.

```
export type Crop = {
   itemType: 'crop'
```